scripts/globals/items/beryllium_bolt_quiver.lua

```diff
@@ -0,0 +1,22 @@
+-----------------------------------
+-- ID: 6428
+-- Beryllium Volt Quiver
+-- When used, you will obtain one stack of Beryllium Bolts
+-----------------------------------
+require("scripts/globals/msg")
+-----------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    local result = 0
+    if target:getFreeSlotsCount() == 0 then
+        result = xi.msg.basic.ITEM_NO_USE_INVENTORY
+    end
+    return result
+end
+
+item_object.onItemUse = function(target)
+    target:addItem(22285, 99)
```

Comment:
For item references, can you please add the appropriate line to `scripts/globals/items.lua`, and then add the require and reference that table in these scripts?

Reply:
and also tick the boxes on the pr template..

Reply:
Sorry, I keep forgetting about those boxes.
I will add them Claywar. Sorry, just noticed.